wallaby.js

```diff
@@ -42,5 +42,7 @@ module.exports = wallaby => {
       }),
         w.testFramework.configure(jestConfig);
     },
+
+    maxConsoleMessagesPerTest: 10000,
```

Comment:
Is this desired to stay in? I think you mentioned something in our walkthrough but I forget.

Reply:
The [default is 100](https://wallabyjs.com/docs/config/overview.html#maximum-console-messages-logged-per-test) which wasn't enough when I was running tests on the recursive doc parser. I can take it out, but it shouldn't hurt anything. I think they leave it at 100 to be less overwhelming, but I found it helpful to not truncate logs.